### ISSUE TITLE
Customize border or qr code at footer

### ISSUE THREAD

**Issue by mezzomokaue** [LibreSign/libresign#3253]
**Describe the bug**
I don't know why, but in some files the QR Code happens to invade the written part of the PDF.

Is there any way to fix or change the layout of the QR Code so this doesn't happen?

![image](https://github.com/LibreSign/libresign/assets/130859376/a3c42fbf-a6e3-4d59-a80b-1225ff11f64f)

 - Browser Chrome
 - LibreSign Version 9.1.2
 - Nextcloud Server Version 29.0.1

**Comment by vitormattos** [comment]
The qrcode need to have a border to work fine. For now isn't possible to customize the border size but could be a good feature.

I recommend to generate a document with an area at footer to put the qrcode.

**Comment by vitormattos** [comment]
If you want, we can implement this feature. To do this we can talk about enterprise support.

**Comment by mezzomokaue** [comment]
How does this corporate support work?

**Comment by vitormattos** [comment]
@mezzomokaue I'm Brazilian too.

You can join to group of [LibreSign at Telegram](https://t.me/LibreSign) and we can talk more about the ways to start a business talk between BR Serviços and LibreCode coop.

**Comment by mezzomokaue** [comment]
Showw

**Comment by rodsmar** [comment]
I've reduced QR MIN SIZE to 30 and fit my word template, looks very well.

@vitormattos Maybe can you add a parameter in config page to adjust the qr size and text in footer.

**Comment by tmlmt** [comment]
I agree this would be a great feature. A temporary quick "fix" could be to reduce from 100 to something like closer to the height of the text itself. Most PDFs get invaded by the huge QR code which is currently added.

> I've reduced QR MIN SIZE to 30 and fit my docx template, looks very well.

Do you mean that you've manually changed this value in the file? https://github.com/LibreSign/libresign/blob/df5923a0fe00210419457dabe4170a6c54efa31a/lib/Handler/FooterHandler.php#L37

**Comment by vitormattos** [comment]
Hi, the signature footer has changed a lot since this issue was opened.

LibreSign now has a configurable signature footer template, including control of the QR code size and a preview of the resulting footer.

Could you test the current version and let us know if the new footer configuration solves the problem you reported here?

If LibreSign is useful for your organization, please consider supporting the project. Community contributions, GitHub Sponsors and our enterprise support help us maintain LibreSign and accelerate new features.
